blueprints/app/files/config/optional-features.json

```diff
@@ -1,3 +1,6 @@
 {
-  "jquery-integration": false
+  "default-async-observers": true,
```

Comment:
Nitpick: can this be moved under application-template-wrapper so they are alphabetical and match what you'd get if you run ember feature:enable default-async-observers on an app that already has the other three optional features?